src/components/PostDetails.tsx

```diff
@@ -132,10 +65,8 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
                     type="button"
                     className="delete is-small"
                     aria-label="delete"
-                    onClick={() => deleteComment(comment.id)}
-                  >
-                    delete button
-                  </button>
+                    onClick={() => dispatch(removeComment(comment.id))}
```

Comment:
Here you are creating a new function on each render. This might lead to performance issues in larger applications. Consider using useCallback hook to memoize the function.